homeassistant/components/tessie/sensor.py

```diff
@@ -51,6 +51,19 @@ class TessieSensorEntityDescription(SensorEntityDescription):
 
 
 DESCRIPTIONS: tuple[TessieSensorEntityDescription, ...] = (
+    TessieSensorEntityDescription(
+        key="charge_state_charging_state",
+        icon="mdi:ev-station",
+        options=[
+            "Starting",
```

Comment:
The options here do not match the keys defined in `strings.json`.

Please ensure that both are the same.

Reply:
Translation keys are case insensitive, the options list is case sensitive, so these do match.

Reply:
Discussed this point internally and we came the the conclusion, that state strings should always be lowercase snake_case.

Reply:
Snake case? does that mean I have to build a translation later from API strings to snake_case strings, making them lower case isnt enough?

Reply:
Ive added a dictionary to convert from the API TitleCase to this new requirement for snake_case